src/views/Settings/Network/Network.vue

```diff
@@ -283,7 +283,7 @@ export default {
       this.startLoader();
       this.$store
         .dispatch('network/saveHostname', modalFormData)
-        .then(this.$store.dispatch('authentication/logout'))
+        .then(() => this.$store.dispatch('authentication/logout'))
```

Comment:
Why this change?

Reply:
Here only after the successful completion of the 'network/saveHostname' action, the 'authentication/logout' action should be dispatched. However, It is immediately invoking this.$store.dispatch('authentication/logout') instead of passing it as a callback function (Even if the hostname fails to update, the page should remain the same).